src/applications/vaos/utils/scrollAndFocus.js

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react';
+import { focusElement } from 'platform/utilities/ui';
+
+export function scrollAndFocus(selector = 'h1,.loading-indicator') {
+  const el = document.querySelector(selector);
+  focusElement(el);
+}
+
+export function useScrollAndFocus() {
+  useEffect(() => {
+    if (window.History) {
+      window.History.scrollRestoration = 'manual';
```

Comment:
Cool, scrollRestoration is new to me! But should we also check to make sure it's a valid property before we set its value?

Reply:
I copied this code over from the current form system code, so it seems to be working as-is in all our multi-step forms.

Reply:
It's probably throwing errors in the console in older browsers since support isn't great, but it won't actually break anything https://caniuse.com/#search=scrollRestoration

Reply:
Ah, could be. I think we had added this because Chrome in particular was trying to scroll the page when the forms were also trying to scroll, causing this bouncing effect. Older browsers didn't have that issue, so it didn't matter that this setting wasn't supported.